### PR TITLE
This completes drop_ops

### PR DESCRIPTION
**Pull request by karthik**
All tests are passing.

### CODE REVIEW

File: R/drop_file_ops.R [hunk 1]
```diff
@@ -1,131 +1,254 @@
...
+        message(sprintf("%s copied to %s", from_path, res$metadata$path_lower))
+        invisible(res)
+      } else {
+        pretty_lists(res)
+        invisible(res)
+      }
     } else {
-      pretty_lists(res)
-      invisible(res)
+      stop("File or folder not found \n")
```
Comment (ClaytonJY) [inline R/drop_file_ops.R]: do you want to error, or return FALSE? can't do both; `stop` prevents `FALSE` from being returned here.

File: R/drop_file_ops.R [hunk 2]
```diff
@@ -1,131 +1,254 @@
...
+        message(sprintf("%s moved to %s", from_path, res$metadata$path_lower))
+        invisible(res)
+      } else {
+        pretty_lists(res)
+        invisible(res)
+      }
     } else {
-      pretty_lists(res)
-      invisible(res)
+      stop("File or folder not found \n")
```
Comment (ClaytonJY) [inline R/drop_file_ops.R]: same issue here; I bet this is why these tests are failing.

File: R/drop_file_ops.R [hunk 3]
```diff
@@ -1,131 +1,254 @@
...
+
+      if (verbose) {
+        res
+      } else {
+        message(sprintf('%s was successfully deleted', res$metadata$path_display))
+        invisible(res)
+      }
     } else {
       # Since file/folder wasn't found, report a stop error
       stop("File not found on current path")
```
Comment (ClaytonJY) [inline R/drop_file_ops.R]: and here

File: R/drop_file_ops.R (hunk 1 above)
Comment (karthik) [reply R/drop_file_ops.R]: Removed `FALSE`

File: R/drop_file_ops.R (hunk 2 above)
Comment (karthik) [reply R/drop_file_ops.R]: Removed `FALSE`

File: R/drop_file_ops.R (hunk 3 above)
Comment (karthik) [reply R/drop_file_ops.R]: Removed `FALSE`

**Pull request merged**